src/components/UI/Modal.js

```diff
@@ -1,37 +0,0 @@
-import { Fragment } from "react";
-import ReactDOM from "react-dom";
-
-import classes from "./Modal.module.css";
```

Comment:
Is there a way to import this with named imports, so we don't need to access `classes.something`. It's nice to know what styles you have available in this file without having to go into an object